src/guide/profiles.md

```diff
@@ -85,7 +85,7 @@ Contains the following mutators:
 * [PowEqual](/guide/mutators.html#Binary-Arithmetic)
 * [ShiftLeft](/guide/mutators.html#Binary-Arithmetic)
 * [ShiftRight](/guide/mutators.html#Binary-Arithmetic)
-* [RoundingFamily](/guide/mutators.html#RoundingFamily)
+* [RoundingFamily](/guide/mutators.html#Round-Family)
```

Comment:
This one seems to work all the same though, for some reason.

Reply:
True, there is some logic that redirects to a similar fragment, if the requested one doesn't exist.